### ISSUE TITLE
Learning center search error message experience

### ISSUE THREAD

**Issue by jenniferlee-dsva** [department-of-veterans-affairs/va.gov-team#14478]
## Learning center search error message experience

**User story:**
As a veteran, I want to know clearly what I should do if my search brings up 0 results. 

  - User enters a search query on the LC search bar using the default "Resources and support" selection. 
  - The user is taken to the results page. 
  - The user receives `null` results. 
  - The user should receive a message telling them we couldn't find resources and support articles matching their query, and that they should try refining their search phrase or expanding their search by searching all of VA.gov. 
- Message text and display should be reusable for users on mobile. 

**AC:**
- [x] Content team reviews draft message and provides final text. [It should not be too long - it needs to work in a mobile context.]
- [x] TBD - Designer provides recommendation and wireframe on how message scenario should appear on desktop and mobile if needed for FE implementation. 
- [ ] FE implementation and validation. 

**Draft text for content review:** 
No resources and support articles were found for "**[user's query]**." Try refining your phrase or use the **All VA.gov** search option.

**ETA:**
We need this before Tze begins QA.

**Comment by andaleliz** [comment]
Is the `tbd` for design on this ticket up to @kelsonic and @ncksllvn to determine if this is needed?

Also, I suggested this before, but how about linking the text referencing `search all va.gov with their query? Then they don't even have to think about finding the "all va.gov" option if they hadn't noticed it. 
 
If someone searched for "service dogs" for example:
No resources and support articles were found for service dogs in the learning center. Try refining your phrase or [search all of VA.gov for service dogs](https://www.va.gov/search/?query=service+dogs).

I think it'd be pretty simple to pass their query into the URL of the error message.

**Comment by jenniferlee-dsva** [comment]
Great suggestion @andaleliz - I love it. RE your first question, I think this error state would be good to have a visual for Tze's QA purposes.

**Comment by andaleliz** [comment]
Here's a visual for review - I'll add it to the Mural board once we have the final copy

![image](https://user-images.githubusercontent.com/55992116/95594509-09372080-0a19-11eb-8ea4-50fd63f1f11e.png)

**Comment by RLHecht** [comment]
@bethpottsVADEPO and I just worked on the content and came up with a more plain language approach for the messaging:

It should read as follows:

No resources and support articles were found for "sandwiches." Try rewording what you're looking for, or search all of VA.gov for sandwiches.

**Comment by andaleliz** [comment]
Sounds good @RLHecht. I'll update the prototype. No bolding?

**Comment by RLHecht** [comment]
@andaleliz Sorry, keep the bolding the way you had it in the screenshot above. I was focused on the wording!

**Comment by andaleliz** [comment]
[Added this screen to the Mural board](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1588167553516/2773c854e8ff3a725a5e3ee03272b78e8519f78b?wid=0-1602257201953)

**Comment by RLHecht** [comment]
@andaleliz Two more changes:

1. The period should go inside the quotes like this: "sandwiches." 
2. There should be quotes in the second sentence around what's being searched too. 

No resources and support articles were found for "**sandwiches**." Try rewording what you're looking for, or search all of VA.gov for "**sandwiches**."

FYSA @bethpottsVADEPO

**Comment by RLHecht** [comment]
@andaleliz As discussed in our slack convo, the new, new content should read as follows:

We didn't find any resources and support articles for "**sandwiches**." Try rewording your search, or [view expanded search results that include all of VA.gov](https://www.va.gov/search/?query=sandwiches).

Thank you @bethpotts and @DanielleThierryUSDSVA for the extra plain language and user sensitivity help!

**Comment by andaleliz** [comment]
@RLHecht Thanks! Mural board screenshot has been updated.  

Will wait for @rtwell to weigh in about the mobile search button before closing this issue.

**Comment by jenniferlee-dsva** [comment]
Could we use fewer words in the message text? It's so hard to track meaning quickly when these simple messages get long. 

We didn't find any resources and support articles for "sandwiches."  Try rewording your search or [view results for all of VA.gov](link.com).

Also - it's okay to use "view"?  I thought we had to avoid "view" now? (That's why we had to change 'View all articles in:...") I'm okay using 'view,' but just confirming, so we are not pinged later for it.

**Comment by bethpottsVADEPO** [comment]
I'd like to go with this (still too long?):
We didn't find any resources and support articles for "sandwiches." Try rewording your search or [expand your results to include all of VA.gov].

**Comment by jenniferlee-dsva** [comment]
It isn't making it any shorter and tbh,  the 'expand your results to include...' is a bit complicated and hard to follow. 

Let's go with: ` ... or view results from all of VA.gov.`

**Comment by andaleliz** [comment]
I think the word `view` still implies that someone has sight. Not totally sure if we'll get dinged for that but throwing some other ideas out there that may be more inclusive: 

`...check results...`
`...review results...`
`...explore results...`
`...browse results...`
`...get results...`

**Comment by bethpottsVADEPO** [comment]
We are trying to avoid "view" so how about this?:
Try rewording your search or [expand your results to all of VA.gov].

**Comment by bethpottsVADEPO** [comment]
We could also just link "expand your results" like this:
Try rewording your search or [expand your results] to all of VA.gov.

**Comment by jenniferlee-dsva** [comment]
I can't tell you why but the "expand" vocabular is not sitting quite right, and it doesn't seem that easy a word. (We're trying to hit 5th grade reading level.) 

But we're getting close! What about - this is both shorter and I think gets to your desired plain language-ness? 

We didn't find any resources and support articles for "sandwiches." Try using different words or [search all of VA.gov](link.com).

**Comment by bethpottsVADEPO** [comment]
Works for me! Let's move forward with that. @andaleliz

**Comment by andaleliz** [comment]
The mural board has been updated with this latest revision.

As an aside, I saw a comment in there that said "See GH comment @liz" from an **Anonymous Squid** 3 days ago.  Unfortunately, Mural doesn't alert anyone when comments are made, so it's not a great way to communicate to the team.  We should probably stick to GH or Slack to make sure comments are seen.

**Comment by brianalloyd** [comment]
@ncksllvn and @kelsonic I believe we're at the 'FE implementation and validation' element for this and just wanted to surface it to the top as a requirement as this is needed in support of QA testing.  Thanks.

**Comment by jenniferlee-dsva** [comment]
Validated, and closing.
